app/modules/entities/components/layouts/author_display.svelte

```diff
@@ -5,11 +5,17 @@
   import { imgSrc } from '#lib/handlebars_helpers/images'
   import getBestLangValue from '#entities/lib/get_best_lang_value'
 
-  export let entityData
+  export let entityData = {}, claimValue
 
-  const { labels, claims, uri, image } = entityData
-  const label = getBestLangValue(app.user.lang, null, labels).value
-  const url = `/entity/${uri}`
+  const { labels, claims = {}, uri, image = {} } = entityData
+  let url, label
+  if (uri) {
+    url = `/entity/${uri}`
+    label = getBestLangValue(app.user.lang, null, labels).value
+  } else {
+    url = `/search?q=${label}`
```

Comment:
The URL could include a `!a` filter to display only authors
```suggestion
    url = `/search?q=!a ${label}`
```

Reply:
done in 76842fb13